ios/RNMagicScript/components/UiNodes/UiImageNode.swift

```diff
@@ -66,6 +69,11 @@ import SceneKit
         planeGeometry = SCNPlane(width: 1.0, height: 1.0)
         planeGeometry.firstMaterial?.lightingModel = .constant
         planeGeometry.firstMaterial?.diffuse.contents = UIColor.init(white: 1, alpha: 0)
+        // Use .clampToBorder instead of .clamp if you want to remove visual "artifacts" that are
```

Comment:
why not use `.clampToBorder` ?

Reply:
Because we will have black sides like in this picture:
![clampToBorder](https://user-images.githubusercontent.com/199881/78254184-d1916e00-74f5-11ea-9746-11935b9fa68e.png)